README.md

```diff
@@ -1,2 +1,106 @@
-# docs
-documentation
+# Using of pax.express
+
+The pax.express service is used for the delivery of firmware to esp32 paxcounter devices. This is done by using the OTA feature of 
+the paxcounter project. Our fork of the repo can be found [here](https://github.com/paxexpress/ESP32-Paxcounter). The pax.express cli can be found [here](https://github.com/paxexpress/paxexpress-cli). Both repositories are used in the discussed setup guide later one. Feel free to clone themen for use.
+
+## Setup guide
+### Tooling 
+
+Please clone the [paxexpress-cli](https://github.com/paxexpress/paxexpress-cli)  to your filesystem. We use poetry for python package management so please also intall [poetry](https://python-poetry.org/docs/#installation).
+
+Using poetry change dirctory to the repo's root directory and call:
+```
+poetry env use python3.7
+
+poetry shell
+
+poetry install
+```
+
+after this you should get the cli's command options by using
+
+```
+paxexpress --help
```

Comment:
It is unclrear from the docs that this command only works after calling `poetry shell` beforehand.

Maybe use `poetry run` instead?